src/AWS.Deploy.Recipes/CdkTemplates/BlazorWasm/Generated/Recipe.cs

```diff
@@ -0,0 +1,186 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+using Amazon.CDK;
+using Amazon.CDK.AWS.CloudFront;
+using Amazon.CDK.AWS.CloudFront.Origins;
+using Amazon.CDK.AWS.S3;
+using Amazon.CDK.AWS.S3.Deployment;
+using AWS.Deploy.Recipes.CDK.Common;
+using BlazorWasm.Configurations;
+
+// This is a generated file from the original deployment recipe. It is recommended to not modify this file in order
+// to allow easy updates to the file when the original recipe that this project was created from has updates.
+// To customize the CDK constructs created in this file you should use the AppStack.CustomizeCDKProps() method.
+
+namespace BlazorWasm
+{
+    using static AWS.Deploy.Recipes.CDK.Common.CDKRecipeCustomizer<Recipe>;
+
+    public class Recipe : Construct
+    {
+        public Bucket? ContentS3Bucket { get; set; }
+
+        public BucketDeployment? ContentS3Deployment { get; set; }
+
+        public Distribution? CloudFrontDistribution { get; set; }
+
+        public string AccessLoggingBucket { get; } = "AccessLoggingBucket";
```

Comment:
This is basically constant. I do this as instance properties instead of regular constants because I want users to be able to use the pattern of using `nameof` in the calls to `InvokeCustomizeCDKPropsEvent` and `CustomizeCDKProps` callback.